Notification Plugin docs (#22)
breez/breez-sdk-liquid-docs: src/notifications/ios_service.md

# Setting up the Notification Service Extension

> ## Add a Notfication Service Extension target

Reviewer: typo
```suggestion
## Add a Notification Service Extension target
```

In Xcode add a new Target to your application:
1. In the menu `File > New > Target...`
2. Select `Notification Service Extension`
3. Give the name `NotificationService`

This should create a new target with a bundle identifier like `com.example.application.NotificationService` and a folder in your application called `NotificationService`.

## Identifiers

To share data between the main application and service extension targets, you need to add a common `app group` to the main application identifier and new service extension identifier.  Go the [Apple Developer](https://developer.apple.com/account/resources/identifiers/list/applicationGroup) portal, in the Identifiers section select the App Groups dropdown to the right, then:
1. Click `+` to add an App Group
2. Select `App Groups` from the list

> 3. Give the App Group a description and identifier, like `group.com.example.application`

Reviewer: I think it's better to use a more formal language such as `such as` instead of "like <-example->" on this page :)

> 3. Give the App Group a description and identifier, like `group.com.example.application`

Author: ```suggestion
3. Give the App Group a description and identifier, such as `group.com.example.application`
```

### Update Identifiers
Now add the App Group to your existing main application release and debug Identifiers, in the Identifiers section select the App IDs dropdown to the right, then:
1. Click the Identifier to edit
2. Enable the `App Groups` app service
3. Click `Edit` and select the created app group

### Create Service Extension Identifiers
Create new release and debug Identifiers for the new service extension.
1. Click `+` to add an Identifier
2. Select `App IDs` from the list
3. Give the Identifier a description and identifier the same as the service extension, like `com.example.application.NotificationService`
4. Enable the `App Groups` app service

> 5. Save the Identifier, then edit again it to add the created app group as above

Reviewer: typo
```suggestion
5. Save the Identifier, then edit it again to add the created app group as above
```
